src/index.html

```diff
@@ -7,8 +7,69 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <a href="#" class="logo-link">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
```

Comment:
add empty line only between sibling tags

Reply:
Thank you,
Done